pkg/storage/chunk/cache/groupcache_test.go

```diff
@@ -47,12 +47,10 @@ func TestGroupCache(t *testing.T) {
 }
 
 func setupGroupCache() (*GroupCache, error) {
-	return NewGroupCache(
-		&mockRingManager{},
-		&server.Server{HTTP: mux.NewRouter()},
-		log.NewNopLogger(),
-		nil,
-	)
+	return NewGroupCache(&mockRingManager{}, GroupCacheConfig{
```

Comment:
nit: isn't it cleaner to keep the previous layout with an indented section for the config parameters?